File: app/src/main/java/com/prashant/masterbuddy/ws/VolleyCallback.java
package com.prashant.masterbuddy.ws;

/**
 * Created by tanmay.agnihotri on 1/5/18.
 */

public interface VolleyCallback {

    public void starting();
    public void onSuccess(Object objectFromJson);
    public void onSuccess(boolean isSuccess);
    public void onFailure();

}

File: app/src/main/java/com/prashant/masterbuddy/ws/VolleyCallback.kt
package com.prashant.masterbuddy.ws

/**
 * Created by tanmay.agnihotri on 1/5/18.
 */

interface VolleyCallback {

    fun starting()
    fun onSuccess(objectFromJson: Any?)
    fun onSuccess(isSuccess: Boolean)
    fun onFailure()

}
